### PR TITLE
Add doc build to travis

### PR DESCRIPTION
**Pull request by hmacdope**
Add a doc build on travis.

### CODE REVIEW

**Comment by hmacdope** [comment]
Hey @orbeckst I am far from a travis expert, but I think this might address #58?

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,14 @@ branches:
   only:
     - master
 
+matrix:
+  fast_finish: true
+  include:
+    - env: NAME="Docs"
+           MAIN_CMD="cd docs && make html"
```
Comment (orbeckst) [inline .travis.yml:17]: This works. However, I think we tend to use the OS-independent
```suggestion
           MAIN_CMD="python setup.py build_sphinx"
```
nowadays.

File: .travis.yml [hunk 2]
```diff
@@ -10,6 +10,14 @@ branches:
...
+           PYTHON_VERSION=3.6
```
Comment (orbeckst) [inline .travis.yml:18]: 3.6 is already getting old...
```suggestion
           PYTHON_VERSION=3.7
```

**Review by orbeckst — CHANGES_REQUESTED** [review]
Yes, that's good in principle but let me be a terrible stickler and throw in two changes.

File: .travis.yml (hunk 1 above)
Comment (hmacdope) [reply .travis.yml:17]: aha unfortunately #61. I am having serious trouble getting `build sphinx` to work. :(

File: .travis.yml (hunk 2 above)
Comment (hmacdope) [reply .travis.yml:18]: Okay sure, we will switch to 3.7, but should deploy the docs off the 3.7 build then also. Ill fix it up.

**Review by orbeckst — APPROVED** [review]
Ok, then we use this and update when #61 is fixed.

**Pull request merged**